### ISSUE TITLE
tcp6 metrics not available via Prometheus output

### ISSUE THREAD

**Issue by hangongithub** [google/cadvisor#2066]
When I try to get tcp connections in cAdvisor's prometheus metrics URL I get:

`container_network_tcp_usage_total{id="/docker/dcf4242fc72230339beba6e3e4ee8cad417dfd12cae9661412918eb097d83d40",image="sha256:90da729c4e729c708f9cf297b734aca942cc304859a42435d2f55d08039f31ed",name="telegrafTCP-02429775-59d7-95c7-9c9b-734d9b57154d",tcp_state="established"} 0`

This is false because if i run netstat using nsenter, I get

```
nsenter --target $PID -n netstat -t | grep -i established | wc -l
7
```
However, if I access the JSON api, v2.1 I see the correct numbers:

```
"tcp6": {
            "Established": 8,
            "SynSent": 0,
            "SynRecv": 1,
            "FinWait1": 0,
            "FinWait2": 0,
            "TimeWait": 0,
            "Close": 0,
            "CloseWait": 21,
            "LastAck": 1,
            "Listen": 2,
            "Closing": 0
          },
```

My suspicious is that the prometheus output is leaving out tcp6 by mistake. I've search for any metrics outputted by cadvisor that looks like it might be tcp6 but haven't found any.

**Comment by hangongithub** [comment]
It looks like NetworkStats has the numbers:

https://github.com/google/cadvisor/blob/49e7c7ead43c3a5ac1f90d9b184b736e57ca1abf/info/v2/container.go

type NetworkStats struct {
	// Network stats by interface.
	Interfaces []v1.InterfaceStats `json:"interfaces,omitempty"`
	// TCP connection stats (Established, Listen...)
	Tcp TcpStat `json:"tcp"`
	// TCP6 connection stats (Established, Listen...)
	Tcp6 TcpStat `json:"tcp6"`
	// UDP connection stats
	Udp v1.UdpStat `json:"udp"`
	// UDP6 connection stats
	Udp6 v1.UdpStat `json:"udp6"`
}

But the prometheus output doesn't try to output it at all:

https://github.com/google/cadvisor/blob/f6334480669bf68c181350ea2fef310e183e31c6/metrics/prometheus.go

It only calls s.Network.Tcp without calling s.Network.Tcp6

**Comment by dashpole** [comment]
Yeah, is probably just hasn't been added yet.  As long as it is not output when the metric is disabled, I think it is appropriate to add.
